app/policies/application_policy.rb

```diff
@@ -42,6 +42,10 @@ def admin?
     user.admin?
   end
 
+  def inactive?
+    user.inactive?
```

Comment:
Removed completely. Wasn't in use anywhere after closer review. PCMOs don't have ability to edit users. We can easily set this up later again when we flush out more of the PCMO admin-like abilities.